.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish to Docker Hub
+
+on:
+  push:
+    branches:
+      - main
+      - objetivo-5
```

Comment:
Mismo error que @mapachana, mismo comentario. Si habéis llegado a este error de forma independiente, OK. Si no, os rogaría que trabajárais de forma autónoma.

Reply:
Añadí la rama `objetivo-5` porque ahora mismo solo estoy realizando cambios en esa rama. Al principio puse solo `main` y la Github Action no se ejecutaba por eso mismo.
Voy a investigar cómo hacerlo si no.